src/main/java/com/epam/aidial/core/config/ApiKeyData.java

```diff
@@ -14,6 +14,7 @@ public class ApiKeyData {
     private String traceId;
     private String spanId;
     private Set<String> attachedFiles = new HashSet<>();
+    private String sourceDeployment;
```

Comment:
can you write some comments what fields related to what?

Reply:
fixed